browser/ui/views/location_bar/onion_location_view.cc

```diff
@@ -52,7 +52,7 @@ void OnTorProfileCreated(GURL onion_location,
   if (!browser)
     return;
   content::OpenURLParams open_tor(onion_location, content::Referrer(),
-                                  WindowOpenDisposition::NEW_FOREGROUND_TAB,
```

Comment:
This is the leftover of https://github.com/brave/brave-core/pull/7713, "Open in Tor" and auto redirect should have same window disposition